racadm.py

```diff
@@ -133,28 +141,29 @@ def logout(self):
         Close the session so we are not stealling spaces
         '''
         #this dosn't work because 'Closing the executing session is not permitted.'
-        #return self.basic_command('closessn -i {}'.format(self.session_id))
+        # return self.basic_command('closessn -i {}'.format(self.session_id))
 
     def login(self):
         '''
         Login to the drac server and get a cookie. Use details pased during initialisation
         '''
-        if 'sid' in self.session.cookies and self.session.cookies['sid'] > 0:
-            self.logger.debug('We already have a session')
-            self.login_state = 'OK'
-            return True
+        # if 'sid' in self.session.cookies and self.session.cookies['sid'] != 0:
+            # self.logger.debug('We already have a session')
+            # self.login_state = 'OK'
+            # return True
```

Comment:
Session is not valid,when second call class function.
I suspect the login's problem.
 so code annotation, it works.
but why this problem come? the dell idrace server setting's problem?
thank you